CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# 12.0.0 - May 17, 2024
+
+- Update MSRV to Rust 1.56.1 [#639](https://github.com/rust-bitcoin/rust-miniscript/pull/639)
+- Drop the `Property` trait entirely [#652](https://github.com/rust-bitcoin/rust-miniscript/pull/652)
+- Improve compiler logic when deciding between conjunctions and multi/multi_a [#657](https://github.com/rust-bitcoin/rust-miniscript/pull/657)
+- Several locktime improvements [#654](https://github.com/rust-bitcoin/rust-miniscript/pull/654)
+- Derives `Hash` for pub items [#659](https://github.com/rust-bitcoin/rust-miniscript/pull/659)
+- Introduce `Threshold` type [#660](https://github.com/rust-bitcoin/rust-miniscript/pull/660), 
+  [#674](https://github.com/rust-bitcoin/rust-miniscript/pull/674),
+  and [#676](https://github.com/rust-bitcoin/rust-miniscript/pull/676)
+- Upgrade `bech32` dependency to `v0.11.0` [#661](https://github.com/rust-bitcoin/rust-miniscript/pull/661)
```

Comment:
Either this version is non code, i.e. 0.11.0 or the MSRV Rust version above should be as code (backticks) for consistency